runtime/v2/runc/container.go

```diff
@@ -139,24 +137,7 @@ func NewContainer(ctx context.Context, platform stdio.Platform, r *task.CreateTa
 	}
 	pid := p.Pid()
 	if pid > 0 {
-		var cg interface{}
-		if cgroups.Mode() == cgroups.Unified {
-			g, err := cgroupsv2.PidGroupPath(pid)
-			if err != nil {
-				logrus.WithError(err).Errorf("loading cgroup2 for %d", pid)
-				return container, nil
-			}
-			cg, err = cgroupsv2.LoadManager("/sys/fs/cgroup", g)
-			if err != nil {
-				logrus.WithError(err).Errorf("loading cgroup2 for %d", pid)
-			}
-		} else {
-			cg, err = cgroups.Load(cgroups.V1, cgroups.PidPath(pid))
-			if err != nil {
-				logrus.WithError(err).Errorf("loading cgroup for %d", pid)
-			}
-		}
-		container.cgroup = cg
+		newContainerCgroup(pid, container)
```

Comment:
Are these changes necessary since runu has an out-of-tree shim?  The runj shim doesn't import this package, for example.

Reply:
I'm going to investigate this too if we can avoid importing (not done yet), but I guess with this diff the out-of-tree shim can reduce the duplicated portion of code.

Reply:
Now I spend some time to make this hunk of patch eliminated and successfully removed from this PR by moving portion of code to (out-of-tree) shim implementation.
(I referred the runj shim code).

So this diff is no longer exist in this patchset.  Thanks for the suggestion.